cmd/host-to-ip.go

```diff
@@ -33,19 +32,7 @@ func (n byLastOctet) Less(i, j int) bool {
 }
 
 // sortIPsByOctet - returns a reverse sorted list of hosts based on the last octet value.
-func sortIPsByOctet(ips []string) error {
-	var nips []net.IP
-	for _, ip := range ips {
-		nip := net.ParseIP(ip)
-		if nip == nil {
-			return fmt.Errorf("Unable to parse invalid ip %s", ip)
-		}
-		nips = append(nips, nip)
-	}
+func sortIPsByOctet(ips []net.IP) {
 	// Reverse sort ips by their last octet.
-	sort.Sort(sort.Reverse(byLastOctet(nips)))
-	for i, nip := range nips {
-		ips[i] = nip.String()
-	}
```

Comment:
There are two reasons for I reason I sent this PR.
1. We should standard use ``net``  package functions like ``net.ParseCIDR`` to parse network address as opposed to using functions from ``strings`` package.
2. The existing code, does one too many of string --> net.IP --> string, once for address validation and once for sorting.